docs/manual/java/guide/advanced/IntegratingNonLagom.md

```diff
@@ -26,12 +26,18 @@ Of course, you will also need to add a dependency to the API project that you ha
 
 The Lagom integration client provides [`LagomClientFactory`](api/index.html?com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.html) creating Lagom client services.  This factory creates and manages thread pools and connection pools, so it's important to manage its lifecycle correctly in your application, ensuring that you only create one instance of it, and to shut it down when you're finished with it.
 
-The factory can be instantiated by invoking the static [`create`](api/index.html?com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.html#create-java.lang.String-java.lang.ClassLoader-) method, for example:
+The factory can be instantiated by invoking the static [`create(String, ClassLoader)`](api/index.html?com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.html#create-java.lang.String-java.lang.ClassLoader-) or [`create(String, ClassLoader, ActorSystem, Materializer)`](api/index.html?com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.html#create-java.lang.String-java.lang.ClassLoader-akka.actor.ActorSystem-akka.stream.Materializer-) methods:
+
+The first [`create(String, ClassLoader)`](api/index.html?com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.html#create-java.lang.String-java.lang.ClassLoader-) variant should be used in applications that do not have a running `ActorSystem`. Internally, the `LagomClientFactory` will create a dedicated `ActorSystem` and `Materializer` to be used by the clients. Both will managed by the `LagomClientFactory` and will be shutdown when the factory is closed.
 
 @[create-factory](code/docs/advanced/IntegratingNonLagom.java)
```

Comment:
I think the code snippet should go immediately after the `:` in line 29 and the paragraph explaining when to use `create(String, ClassLoader)` should go after that.

Reply:
Alternatively these two examples should be an `<ul>` so the `:` in line 29 still makes sense.

Reply:
Yeah, I think this whole section needs some love. I will reorganise it.